packages/components/src/font-size-picker/stories/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import FontSizePicker from '../';
+
+export default { title: 'FontSizePicker', component: FontSizePicker };
+
+const FontSizePickerWithState = ( { withSlider, disableCustomFontSizes, ...props } ) => {
+	const [ fontSize, setFontSize ] = useState( 16 );
+	const fontSizes = [
+		{
+			name: 'Small',
+			slug: 'small',
+			size: 12,
+		},
+		{
+			name: 'Medium',
+			slug: 'medium',
+			size: 16,
+		},
+		{
+			name: 'Big',
+			slug: 'big',
+			size: 26,
+		},
+	];
+	const fallbackFontSize = 16;
+
+	return (
+		<FontSizePicker
+			{ ...props }
+			fontSizes={ fontSizes }
+			fallbackFontSize={ fallbackFontSize }
+			value={ fontSize }
+			onChange={ ( newFontSize ) => setFontSize( newFontSize ) }
+			withSlider={ withSlider }
+			disableCustomFontSizes={ disableCustomFontSizes }
```

Comment:
You don't need to include `withSlider` or `disableCustomFontSize` in the `FontSizePickerWithState` because you already have the spread `...props` which will include any passed in properties already.

Reply:
I have pushed a fix for this, I also simplified `onChange={ ( newFontSize ) => setFontSize( newFontSize ) }` to `onChange={ setFontSize }` following a recommendation in another PR